src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -478,6 +478,40 @@ public void UpdateAutoEvoResults(string results, string external)
         externalEffectsLabel.Text = external;
     }
 
+    public void SetRigiditySliderTooltip(float rigidity)
+    {
+        var rigiditySliderTooltip = GetNode("CellEditor/Tooltips/rigiditySlider")
+            .GetNode("MarginContainer/VBoxContainer/VBoxContainer");
+        var healthChangeLabel = rigiditySliderTooltip.GetNode<Label>("HBoxContainer2/Label");
+        var mobilityChangeLabel = rigiditySliderTooltip.GetNode<Label>("HBoxContainer/Label");
```

Comment:
I think it would be better to have a NodePath configured in the editor, instead of these very complex paths here, as the editor node paths get automatically updated if someone edits the GUI.